tgui/packages/tgui/interfaces/SpecMenu.js

```diff
@@ -0,0 +1,99 @@
+import { useBackend } from "../backend";
+import { Box, Button, Flex, Section, Divider } from "../components";
+import { Window } from "../layouts";
+
+export const SpecMenu = (props, context) => {
+  return (
+    <Window resizable theme="nologo">
+      <Window.Content>
+        <Flex justify="space-around">
+          <HemoMenu />
+          <Divider vertical={1} />
+          <UmbrMenu />
+          <Divider vertical={1} />
+          <GarMenu />
+        </Flex>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const HemoMenu = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    subclasses,
+  } = data;
+  return (
+    <Flex.Item grow={1} basis="33%">
+      <Section title="Hemomancer">
+        <p>Focuses on blood magic and the manipulation of blood around you.</p>
```

Comment:
This title section looks better if it's more visible, rather than being the same size as the rest of the text around it. Although you will need to change the window size if you do this, since it's pushing everything downwards.
![image](https://user-images.githubusercontent.com/57483089/132144322-384d9d98-d8cb-42b8-b871-163eb9eb3947.png)
```suggestion
        <h3>Focuses on blood magic and the manipulation of blood around you.</h3>
        <Divider />
```